engine/src/test/java/com/datatorrent/stram/cli/ApexCliTest.java

```diff
@@ -53,16 +56,16 @@
   private static File configFile;
 
 
-  private static AppPackage ap;
-  private static ConfigPackage cp;
-  static TemporaryFolder testFolder = new TemporaryFolder();
-  static ApexCli cli = new ApexCli();
+  private AppPackage ap;
+  private ConfigPackage cp;
+  TemporaryFolder testFolder = new TemporaryFolder();
+  ApexCli cli = new ApexCli();
 
-  static Map<String, String> env = new HashMap<String, String>();
-  static String userHome;
+  Map<String, String> env = new HashMap<String, String>();
+  String userHome;
 
-  @BeforeClass
-  public static void starting()
+  @Before
+  public void starting()
```

Comment:
The merging of the apps deletes the apps from the app-packages. It means that the extracted packages cannot be reused. So setup needs to be done before every test. I do see some operation that can be done at class level.